specs/proposals/oss-release-signing.md

```diff
@@ -0,0 +1,89 @@
+## Proposal for Signing and Verifying OSS Release Assets
+
+### Overview
+
+This document proposes enhancing Notation's capabilities for signing and verifying Open Source Software (OSS) community released assets. It outlines the design for a new vendor-neutral `notation-signer` plugin and introduces a simplified, stateless verification mode for the `notation blob verify` command.
+
+### Problem Statement & Motivation
+
+Notation v2.0.0-alpha.1 introduced blob signing, enabling the signing of assets like GitHub releases. However, several issues remain:
```

Comment:
Suggest listing the existing GitHub issues for justification. It sets the background for reviewers and lets people understand that this proposal aims to resolve existing gaps.

- Sign Notation CLI release assets with Notation GitHub Actions https://github.com/notaryproject/notation/issues/973
- Support using signing key from GitHub Encrypted Secret https://github.com/notaryproject/notation/issues/905
- Signing with local private keys https://github.com/notaryproject/notation/issues/539
- ux: broken default `notation verify` experience: https://github.com/notaryproject/notation/issues/430

Reply:
Nice idea. Updated.